acceleration/fast_inference_tutorial/fast_inference_tutorial.ipynb

```diff
@@ -0,0 +1,635 @@
+{
```

Comment:
<p>Do you think we can also include the <code>.nii.gz</code> benchmark result in the notebook since the original data is <code>nii.gz</code> format.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1948/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Hi @KumoLiu , thanks for the suggestion. `.nii.gz` files have to be decompressed in CPU, thus using GDS may not have acceleration. I added a section to introduces the limitations on each feature, could you help to review the updates? Thanks!